### ISSUE TITLE
dif-template-manager cannot find dif template when using relative configuration file path

### ISSUE THREAD

**Issue by vmaffione** [IRATI/stack#656]
````
+ sudo irati/local/bin/ipcm -c ipcmanager.conf-normal -l DEBUG
+ tee log
379(1435572662)#ipcm (DBG): SIGSEGV handler installed successfully
379(1435572662)#ipcm (DBG): SIGINT handler installed successfully
379(1435572662)#ipcm (DBG): SIGPIPE handler installed successfully
379(1435572662)#ipcm (DBG): Config file is: ipcmanager.conf-normal
379(1435572662)#librina.logs (DBG): New log level: DEBUG
379(1435572662)#librina.common (WARN): Error setting log file, using stdout only
379(1435572662)#librina.nl-manager (DBG): Netlink Manager constructor called, with netlink pid = 1
379(1435572662)#librina.nl-manager (INFO): Netlink socket connected to local port 1 
379(1435572662)#librina.nl-manager (DBG): Generic Netlink RINA family id: 25
379(1435572662)#librina.core (DBG): Initialized Netlink Manager
379(1435572662)#librina.core (DBG): Initialized event queue
379(1435572662)#librina.core (DBG): Started Netlink Message reader thread
379(1435572662)#librina.core (DBG): Waiting for message 1
379(1435572662)#librina.nl-manager (DBG): NL msg TX. Fam: 25; Opcode: 28_IPCM_PRESENT; Sport: 1; Dport: 0; Seqnum: 0; Notification; SIPCP: 0; DIPCP: 0
379(1435572662)#ipcm (DBG): IPC Manager daemon initialized
379(1435572662)#ipcm (DBG):        installation path: /home/vmaffione/irati/local/bin
379(1435572662)#ipcm (DBG):        library path: /home/vmaffione/irati/local/lib
379(1435572662)#ipcm (DBG):        log folder: /home/vmaffione/irati/local/log
379(1435572662)#ipcm (DBG): Starting main I/O loop...
379(1435572662)#ipcm.dif-template-manager (ERR): Failed to open folder ipcmanager.conf-normal
379(1435572662)#ipcm.scripting (DBG): Starting...
379(1435572662)#ipcm (ERR)[apply_configuration]: Cannot find template called default.dif
379(1435572662)#ipcm.scripting (DBG): Finalizing...
379(1435572662)#ipcm.console (DBG): Console starts [fd=4]
````

@edugrasa it seems the dif-template-manager does not support relative paths - default.dif is in the right place, in the same directory as the IPCM configuration file.

**Comment by edugrasa** [comment]
The issue is not specifying a relative path - that is supported (just checked it). The only problem is when the IPC Manager configuration file (and templates) are in the same folder as the ipcm.sh script. Fixing that..

**Comment by edugrasa** [comment]
Fixed by #766
